_g3/tests/test-register-design-setting-meta.php

```
@@ -0,0 +1,121 @@
+
+class RegisterDesignSettingMetaTest extends WP_UnitTestCase {
+
+	/**
+	 * Test that REST API update succeeds with various _lightning_design_setting values.
+	 * さまざまな _lightning_design_setting 値で REST 更新が成功することをテストする。
+	 */
+	public function test_lightning_g3_register_design_setting_meta() {
+
+		// テスト用の固定ページを作成する。
+		// Create a test page.
+		$post_id = $this->factory->post->create(
```

PHPStan (2.1.46)

[error] Access to an undefined property RegisterDesignSettingMetaTest::$factory.
Learn more: https://phpstan.org/blog/solving-phpstan-access-to-undefined-property

(property.notFound)